platforms/access-bridge/src/server.ts

```diff
@@ -66,6 +74,11 @@ export class Server {
       return;
     }
 
+    if (err instanceof Stripe.errors.StripeError) {
```

Comment:
GlobalErrorHandler is getting big enough that it can probably move to its own file. Maybe consider something like a `/pipeline` folder where you can move `endpoints.ts` and `globalErrorHandler.ts` and maybe some other things.